Majasdarbs/04-majasdarbs/bootstrap/04-katrina-tiltina/04-KT-CV-adaptivs-dizains/style.css

```diff
@@ -0,0 +1,438 @@
+    
+    
+}
+
+.card h4{
+    padding: 3rem;
+}
+
+.card-icon{
+    background: rgb(8, 99, 117);
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
+}
+
+
+/*article starts here*/
+
+
+.article {
+    padding:2em 10%;
+    margin: auto;
+    display:inline-block;
+    
+    color:rgb(60, 22, 66);
+    font-family: sans-serif;
+  }
+
+.article h3{
+    color: rgb(8, 99, 117);
+}
+
+    .aTitleArea {
+      padding: 2em 0;
+      margin-bottom: 1em;
+      text-align: center;
+      overflow: hidden;
+      position: relative;
+    }
+      .aTitle {
+        color:rgb(60, 22, 66);
+        font-size: 2.8rem;
+        font-family: "Montserrat", sans-serif;
+      }
+  
+        .aTitle span{
+          display: inline-block;
+          font-size: 1em;
+          position: relative;
+  
+        }
+        .aTitle span:after, .aTitle span:before{
+          content:"";
+          position: absolute;
+          border-bottom:3px solid;
+          width:4em;
+          margin:0 1.5em;
+          bottom:.5em;
+        }
+        .aTitle span:before{
+          right:100%;
+        }
+        .aTitle span:after {
+          left:100%;
+        }
+
+  
+    .aContent {
+        font-size: 2rem;
+        
+    }
+    em{
+        color:lightgrey;
+    }
+  
+   /*project section stats here*/
+
+   .projects {
+    width: 90%;
+    margin: auto;
+    min-height: 90vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    flex-wrap: wrap;
+  }
+  
+  .travelly,
+  .elena {
+    flex: 1 1 40rem;
+  }
+
+.elena h2, .travelly h2{
+    padding: 2rem;
+}
+  
+
+
+   /*contact section starts here*/
+#contact{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+    
+
+}
+
+.form-wrapper,.about-wrapper{
+    text-align: center;
+    padding: 5rem;
+    margin: 80px;
+    
+    
+}
+
+.about-wrapper{
+   flex: 1 2 40px;
+    border-left: 3px solid rgb(60, 22, 66);;
+    
+}
+
+.about-wrapper header img{
+    width: 200px;
+    margin: 10px;
+}
+
+.about-wrapper header h2{
+    margin: 1rem;
+}
+
+.form-wrapper{
+    background-image:linear-gradient(rgba(60, 22, 66,0.5), transparent), url("img/3.png");
+    background-size: cover;
+    
+    color: aliceblue;
+    border-radius: 20px;
+    flex: 1 1 40px;
+    
+}
+
+.form-head{
+    text-align: center;
+    padding: 4rem;
+}
+
+.name-form,.email-form, .text-form{
+    padding: 2rem;
+    font-size: 2rem;
+    
+}
+
+.a-contact{
+    color: black;
+    font-size: 1.3rem;
+}
+
+
+   /*footer section starts here*/
+   footer{
+    color: white;
+    background:rgb(60, 22, 66);
+}
+
+.footer-wrapper{
+    display: flex;
+    padding: 2rem;
+    width:90%;
+    margin:auto;
+    align-items: center;
+    min-height: 10vh;
+    
+}
+
+footer h5{
+    flex: 1 1 40rem;
+    font-size: 1.5rem;
+    font-family: sans-serif;
+    font-weight: lighter;
+}
+  
+@media screen and (max-width: 842px) {
```

Comment:
divide in multiple files.
Responsive styles could be inside in a different css file